### ISSUE TITLE
Request - Exclude connected user, cpu and memory checking

### ISSUE THREAD

**Issue by sanjayhirani** [sacha81/XA-and-XD-HealthCheck#72]
Hi, i would like to exclude the 3 things from running. i have 350 servers and it takes too long to run because of mainly cpu and memory checking on each server.

**Comment by sacha81** [comment]
what you want to exclude? Can you change it in the XML?

**Comment by sanjayhirani** [comment]
yes. xml change has been done but that doesn't stop the script polling it. it only means it doesn't show in the report.

I would like it to not poll for those items at script layer.

**Comment by sanjayhirani** [comment]
Hi, any suggestion as to how to stop the script polling this information?
